src/group/libp2p_group_gossip_server.erl

```diff
@@ -335,7 +340,7 @@ assign_target(WorkerPid, WorkerRef, TargetAddrs, State=#state{workers=Workers, s
                     State
             end;
         _ ->
-            SelectedAddr = mk_multiaddr(lists:nth(rand:uniform(length(TargetAddrs)), TargetAddrs)),
+            SelectedAddr = mk_multiaddr(maybe_lookup_seed_in_dns(TargetAddrs)),
```

Comment:
Isn't this function used to assign both seed and non-seed targets? If so, this is probably not doing the right thing?

Reply:
I'm not sure? It seems to be used by worker type `seed` only so that's why I thought this is the place to modify

Reply:
This should be done here: https://github.com/helium/erlang-libp2p/blob/f4d1d84c8f312555da8dbf3df6c41e064abd2fdb/src/group/libp2p_group_gossip_server.erl#L184 instead (just lookup a DNS seed node here and pass a 1 element list to asign_target).